.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+name: Android CI
+
+on: [push, pull_request]
+
+jobs:
+  apk:
+    name: Generate APK
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Create empty local.properties
+        uses: finnp/create-file-action@1.0.0
+        env:
+          FILE_NAME: "local.properties"
+          FILE_DATA: ""
+      - name: Prepare thirdparty
+        run: bash ./gradlew -b thirdparty_build.gradle downloadAndPatchDjvu downloadAndMakeMupdf
+      - name: Build debug APK
+        run: bash ./gradlew assembleCi --stacktrace -Porion.CIBuild=true
+      - name: Upload APK
+        uses: actions/upload-artifact@v2
+        with:
+          name: OrionViewer
```

Comment:
Or maybe just upload 4 files as is, because they are already zipped with time stamp in name?

Would it work if not specify name?

Reply:
https://github.com/marketplace/actions/upload-a-build-artifact

Default name `artifact` will be used. This task has no control over zip or not to zip.